services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/TormaystarkasteluTormaysServiceITest.kt

```diff
@@ -115,8 +115,8 @@ internal class TormaystarkasteluTormaysServicePGITest : DatabaseTest() {
     @Test
     fun raitiotiet() {
         val geometriaIds = createHankeGeometria()
-        assertThat(tormaysService.maxIntersectingTramByLaneType(geometriaIds))
-            .isEqualTo(TormaystarkasteluRaitiotiekaistatyyppi.JAETTU.value)
+        assertThat(tormaysService.anyIntersectsWithTramLines(geometriaIds)).isTrue()
+        assertThat(tormaysService.anyIntersectsWithTramInfra(geometriaIds)).isTrue()
```

Comment:
Is it enough to check that both happen to be true in that one spot? Should there be tests for negative cases?

Reply:
Added a new geometry that does not intersect with any tormays data. Now there are two test geometries, one for central location (Kaivokatu) and one for out-of-all-infra location (Mustikkamaa).